src/pages/Locator.php

```diff
@@ -212,9 +212,7 @@ public function getShowRadius()
      */
     public function getCategories()
```

Comment:
Is this method still needed if it's no longer altering the relation list?

Reply:
The alteration has been added back in. A typo in a relation array caused the initial problem.